### DISCUSSION TITLE
Firebase services stop working for production iOS app

### DISCUSSION THREAD

**Discussion by Ggayane** [invertase/react-native-firebase#7484] [Q&A] [answered]
Same app same versions, on android everything works fine, on iOS starting from today firebase services stop working, I'm getting errors like:
[firestore/unavailable] The service is currently unavailable. This is a most likely a transient condition and may be corrected by retrying with a backoff.
[auth/internal-error] An internal error has occurred, please try again.

Does anyone know what can be the problem?
At first I though it might be firebase outages, but couldn't find anything in health check dashboard....

**Comment by mikehardy** [comment]
Firebase is very picky about inbound networks, I wonder if you are on a network it does not like or a VPN or similar? I have seen this when I use a VPN on android at times. The backend is shared between iOS and Android, it does not care what type of device originates requests, so I would look at what is different on the network paths between devices

**Reply by Ggayane** [reply]
thank you, the thing is this happens for only my production app even if I run it locally on my pc. the same shared code development app works just fine. 
other users from different network also facing with this issue

**Comment by Ggayane** [comment, marked as answer]
turned out the reason was  there were added api restrictions wrongly...
